apps/web/src/services/iam/store/role-form-store.ts

```diff
@@ -0,0 +1,38 @@
+import { reactive } from 'vue';
+
+import { defineStore } from 'pinia';
+
+export const useRoleFormStore = defineStore('role-form', () => {
+    const state = reactive({
+        form: {
+            name: '',
+            role_type: '',
+            permissions: [] as string[],
+            page_access: [] as string[],
+            role_id: '',
+        },
```

Comment:
nitpick: Since 'form' is already in the store's name, isn't it unnecessary to put it inside a 'form' object?

사소한 건데, 스토어 이름에 form이 들어가니 굳이 form 안에 넣을 필요는 없지 않을까요?🧐